dotalytics_api/client.py

```diff
@@ -1,5 +1,5 @@
 from functools import lru_cache
-
+import asyncio
 import requests
```

Comment:
requests does not support async/await syntax. You can use `httpx` as a drop in replacement for requests. The syntax to use httpx looks like this:
```
async def get_match_history(account_id: str, matches_requested: int = 25) -> match_history.GetMatchHistoryResponse:
    url = base_url + 'IDOTA2Match_570/GetMatchHistory/v1'
    async with httpx.AsyncClient() as client:
        resp = await client.get(url, params={'account_id': account_id, 'key': api_key, 'matches_requested': matches_requested})
        resp.raise_for_status()
        return match_history.GetMatchHistoryResponse(**resp.json())
```
The above block of code does work for the get_match_history function, we should apply this pattern to the other requests calls in this file.

Be sure to update the requirements.txt file to read:
```
fastapi>=0.78.0
httpx>=0.23.0
uvicorn>=0.17.6
pydantic>=1.9.1
```
so that we can install httpx on install of the app.